3_extract/out/GLM_run_summary.Rmd

```diff
@@ -0,0 +1,76 @@
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+options(tidyverse.quiet = TRUE)
+current_date <- Sys.Date()
+```
+---
+title: "Summary of GLM runs"
+output: html_document
+date: `r current_date`
+---
+
+### NLDAS runs
+``` {r, echo = FALSE}
+tar_load(p1_nldas_site_ids)
+tar_load(p1_lake_to_state_xwalk_df)
+tar_load(p3_nldas_glm_uncalibrated_output_feather_tibble)
+tar_load(p1_nldas_dates)
```

Comment:
Instead of `tar_load()` here, I am wondering if you should pass these in to the `params` argument of `tar_render()` (see the bottom example here). Otherwise, I don't think the RMD target will automatically rebuild if any of those upstream targets change. I gave a suggestion for what to change in the `tar_render()` code.

Reply:
I had the same thought, but then [the documentation](https://books.ropensci.org/targets/literate-programming.html#r-markdown-targets) suggests that isn't necessary:
> With the tar_render() function in [tarchetypes](https://github.com/ropensci/tarchetypes), we can go a step further and include report.Rmd as a target in the pipeline. This new targets re-renders report.Rmd whenever fit or hist changes, which means tar_make() brings the output file report.html up to date.
>
> tar_render() is like tar_target(), except that you supply the file path to the R Markdown report instead of an R command. 
>
> When we visualize the pipeline, we see that our report target depends on targets fit and hist. tar_render() automatically detects these upstream dependencies by statically analyzing report.Rmd for calls to tar_load() and tar_read().

But I'm happy to also specify them as `params` if we want that extra security

Reply:
> statically analyzing report.Rmd for calls to tar_load() and tar_read()

Cool! I had not noticed that, so I think we are good here!